WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -479,16 +434,14 @@ private extension NotificationDetailsViewController {
             fallthrough
         case .text:
             return NoteBlockTextTableViewCell.reuseIdentifier()
-        case .comment:
-            return NoteBlockCommentTableViewCell.reuseIdentifier()
         case .image:
             return NoteBlockImageTableViewCell.reuseIdentifier()
         case .user:
             return NoteBlockUserTableViewCell.reuseIdentifier()
         case .button:
             return NoteBlockButtonTableViewCell.reuseIdentifier()
         default:
-            assertionFailure("Unmanaged group kind: \(blockGroup.kind)")
+            wpAssertionFailure("Unmanaged group kind", userInfo: ["kind": "\(blockGroup.kind)"])
```

Comment:
Fancy bumping into this here... I spent some time looking at this and other methods in [`WPInstrumentation.swft`](https://github.com/wordpress-mobile/WordPress-iOS/blob/6a1f355d27bd00c61d5ab570c6bb752b9832f448/WordPress/Classes/Utility/WPInstrumentation.swift) because some of the Core Data code I'm interacting with uses it.

We may need to extract down to a shared layer, but that will not be trivial because of shared state...

https://github.com/wordpress-mobile/WordPress-iOS/blob/6a1f355d27bd00c61d5ab570c6bb752b9832f448/WordPress/Classes/Utility/WPInstrumentation.swift#L25

...and Tracks

https://github.com/wordpress-mobile/WordPress-iOS/blob/6a1f355d27bd00c61d5ab570c6bb752b9832f448/WordPress/Classes/Utility/Logging/WPCrashLoggingProvider.swift#L3-L7

Reply:
It should be possible to inject it:

```
WPInstrumentation.logger = WPLoggingStack.shared // implementing minimal required protocol
```